lib/commands/proxy-helper.js

```diff
@@ -7,7 +7,7 @@ const GET_STATUS_COMMAND = 'getStatus';
 
 let helpers = {}, extensions = {};
 
-helpers.proxyCommand = async function (endpoint, method, body) {
+helpers.proxyCommand = async function (endpoint, method, body, isSessionCommand=true) {
```

Comment:
I don't think we've documented this appropriately in our style guide, but we put spaces around `=` in default parameter values like this.